src/client/pages/Geo/GeoMap/GeoMapMenuStatistics/geoMapMenuStatistics.scss

```diff
@@ -12,3 +12,7 @@
     width: 700px;
   }
 }
+
+.geo-map-alt-message {
+  padding: 0 0 1em 1em;
```

Comment:
we don't use em. px is preferred . use $spacing-XXX

Reply:
Okay, I fixed it here and in the other .scss files from the statistics panel.

Reply:
Okay, I fixed it here and in other .scss files from the statistics panel.